src/shared/bg_misc.cpp

```diff
@@ -1406,7 +1406,7 @@ This is done after each set of usercmd_t on the server,
 and after local prediction on the client
 ========================
 */
-void BG_PlayerStateToEntityState( playerState_t *ps, entityState_t *s, bool snap )
+static void PlayerStateToEntityState( playerState_t *ps, entityState_t *s, bool snap, bool extrapolate, int time )
```

Comment:
Actually, there's an even better solution: to move the code which uses those parameters out of the function, since it's only used by `BG_PlayerStateToEntityStateExtraPolate`. Also, the "snap" thing, it's never used by cgame, and always by sgame, so same here, it can be moved out of this function.
I'm in the process of seeing if there's other stuff I can clean in that stuff, original commits were only about removing huge block of duplicated code, but can improve a bit on top of that.